source/features/show-names.tsx

```diff
@@ -38,22 +45,27 @@ async function init(): Promise<false | void> {
 	);
 
 	for (const usernameEl of usernameElements) {
-		const userKey = api.escapeKey(usernameEl.textContent!);
+		const username = usernameEl.textContent!;
+		const userKey = api.escapeKey(username);
 
 		// For the currently logged in user, `names[userKey]` would not be present.
 		if (names[userKey] && names[userKey].name) {
 			// If it's a regular comment author, add it outside <strong>
 			// otherwise it's something like "User added some commits"
-			const insertionPoint = usernameEl.parentElement!.tagName === 'STRONG' ? usernameEl.parentElement! : usernameEl;
-			insertionPoint.after(
-				' (',
-				<bdo className="css-truncate">
-					<span className="css-truncate-target" style={{maxWidth: '200px'}}>
-						{names[userKey].name}
-					</span>
-				</bdo>,
-				') '
-			);
+			if (compareNames(username, names[userKey].name)) {
+				usernameEl.textContent = names[userKey].name;
```

Comment:
This breaks "copy-pastability" of users. The alternative is to add the spaces and punctuation via non-selectable elements, as suggested in the original issue.

@sindresorhus thoughts? Is copy-pastability important?

Reply:
Maybe it's ok, the real username appears in the hovercard

Reply:
Not so easy to copy though
![Video_2019-06-30_095646](https://user-images.githubusercontent.com/16872793/60397687-86d56680-9b1d-11e9-8d48-474698266bcc.gif)

Reply:
Uhm? The regular username is the same, it's a link so you have to hold `alt` to select it easily